docs/client-development/mandate.rst

```diff
@@ -0,0 +1,520 @@
+
+.. note::
+    This is an experimental feature and it doesn't rely on existing ZGW standards.
+
+
+Cases can be initiated and managed by different parties. They can be individuals, organizations,
+municipality employees, etc. ``Rol`` resource from Zaken API is used to store information
+about such parties.
+
+If the user is authenticated with Digid or eHerkenning, the meta information about it can be
+saved into ``Rol.authenticatieContext``. The user can initiate the case on their own behalf,
+but it's also possible to represent another party. Both Digid and eHerkenning support such mandates.
+
+.. image:: _assets/machtiging.png
+    :width: 100%
+    :alt: Digid and eHerkenning authorized parties with and without mandates.
+
+
+In the schema you can see all possible options for mandates for users:
+
+1. An individual (``natuurlijk_persoon``) is authorized with Digid and initiates their own cases
+2. An individual (``natuurlijk_persoon``) is authorized with Digid and initiates the case on
+   behalf of another individual (Digid machtiging)
+3. An employee of the organization (``niet_natuurlijk_persoon`` or ``vestiging``) is authorized with
+   eHerkenning and initiates their own cases
+4. An employee of the organization (``niet_natuurlijk_persoon`` or ``vestiging``) is authorized with
+   eHerkenning and initiates the case on behalf of another individual (eHerkenning bewindvoering)
+5. An employee of the organization (``niet_natuurlijk_persoon`` or ``vestiging``) is authorized with
+   eHerkenning and initiates the case on behalf of another organization (eHerkenning ketenmachtiging)
+
+
+Open Zaak ``/zaken/api/v1/rollen`` endpoint supports all these options. Here are examples how to use it.
+
+Recipies to create rollen with mandates
+---------------------------------------
+
+General rules
+^^^^^^^^^^^^^
+
+* all information about mandates is stored in ``Rol.authenticatieContext``
+* mandates are supported only for ``natuurlijk_persoon``, ``niet_natuurlijk_persoon`` and
+  ``vestiging`` values of ``Rol.betrokkeneType``
+* only ``digid`` and ``eherkenning`` are supported as sources for mandates
+* an authorizee (``gemachtigde``) and a representee (``machtiginggever``) are defined by
+  ``Rol.indicatieMachtiging`` attribute. If it's blank, that means that the party initiates
+  their own case.
+* for eHerkenning cases the employee details should be added to ``Rol.contactpersoonRol``
+  attributes.
+
+Validation rules
+^^^^^^^^^^^^^^^^
+
+* if ``representee`` is provided in ``authenticatieContext``, then:
+    * ``indicatieMachtiging`` **MUST** be set to "gemachtigde"
+    * ``mandate`` **MUST** be provided in the ``authenticatieContext``
+
+* if ``betrokkeneType`` is ``niet_natuurlijk_persoon``, then ``Rol.authenticatieContext.source``
+  **MUST** be set to "digid"
+
+* if ``betrokkeneType`` is ``natuurlijk_persoon`` or ``vestiging``, then
+  ``Rol.authenticatieContext.source`` **MUST** be set to "eherkenning"
```

Comment:
1 and 2 above suggest digid should be `natuurlijk_persoon` and 3, 4, 5 eherkenning  should be `niet_natuurlijk_persoon`

Is this wrong or am I misinterpreting?

Reply:
The examples also show this

Reply:
bear with me since I didn't look at the entire context, but eherkenning bewindvoering is on a personal title (BSN) so that implies natuurlijk persoon. That's the case where an employee of a company is managing the affairs of a person (by court mandate?)

Reply:
@Coperh Yes, sorry, you are right my brain short circuited. It should be the other way round. 

@sergei-maertens when "eherkenning bewindvoering" is the case then the authorizee is an organization (`niet_natuurlijk_persoon` or `vestiging`) and a representee is an individual (`natuurlijk_persoon`), so we expect the creation of two rollen, right? But the rol with a representee would have `Rol.authenticatieContext` = `None`, so this validation won't be applied there. 
Maybe I should change it to something like:
```
* if ``betrokkeneType`` is ``niet_natuurlijk_persoon`` or ``vestiging`` and ``Rol.authenticatieContext`` is provided, then
  ``Rol.authenticatieContext.source`` **MUST** be set to "eherkenning"
```

Is it more clear this way?

Reply:
@Coperh I fixed, thank for catching it